docs/resources/Monster.md

```diff
@@ -0,0 +1,185 @@
+# Monster Resource
+
+Monsters ID's here.
+
+###### Monster IDs
+
+|           ID          | Hex |
+| ----------------------|-----|
+|  UNK                  | 00  |
```

Comment:
what is this?

Reply:
I don't know, I never actually saw this in any quest nor never used it.

Reply:
better open an issue when i merge over it